src/test/java/nodejs/flows/albums/buyAlbum.feature

```diff
@@ -0,0 +1,28 @@
+Feature: buying albums
+
+  Background:
+    * url urlNodeJS
+
+    * def errorSchema = read('classpath:nodejs/schemas/error.json')
+
+    * def auth = callonce read('classpath:nodejs/flows/auth/loginRole.feature@regular')
+    * def token = auth.token
+    * def userId = auth.userId
+
+    * def albums = call read('classpath:nodejs/flows/albums/albums.feature@all')
+    * def albumId = albums.randomAlbumId
+
+  Scenario: buy a random new album
+    * call read('classpath:nodejs/flows/albums/albums.feature@buy')
+    * call read('classpath:nodejs/flows/users/userAlbums.feature')
+    * match userAlbums contains albumId
```

Comment:
Why this 3 variables isn´t declarated in the background?, if all out of the same call read?. I think that could go off scenario.